packages/core/scss/components/grid/_theme.scss

```diff
@@ -91,6 +91,10 @@
             background-color: $kendo-grid-hover-bg;
         }
 
+        .k-table-td-row-span {
+            box-shadow: $kendo-grid-cell-row-span-shadow;
```

Comment:
Is this a generic enhancement for the Table ? If so, the respective styles should be part of the Table module.

Reply:
This is rather enhancement for the Grid component, so in my opinion these styles should be part of the Grid module.

Reply:
We further discussed this with @TeyaVes and @zhpenkov and agreed to move these styles to the Table component